plugins/woocommerce-blocks/packages/components/radio-control-accordion/index.tsx

```diff
@@ -50,7 +67,13 @@ const RadioControlAccordion = ( {
 				const checked = option.value === selected;
 				return (
 					<div
-						className="wc-block-components-radio-control-accordion-option"
+						className={ classnames(
+							'wc-block-components-radio-control-accordion-option',
+							{
+								'wc-block-components-radio-control-accordion-option--checked-option-highlighted':
```

Comment:
I'm not sure I understand,  sorry. This _is_ the parent. Inside this component an `Option` and a description is rendered, so the parent is the one that needs the border, rather than the children.

Maybe I misunderstood what you mean though?